src/commands/Bunnies.js

```diff
@@ -0,0 +1,50 @@
+const { Command } = require('@dyno.gg/dyno-core');
+const superagent = require('superagent');
+
+
+class Bunnies extends Command {
+    constructor(...args) {
+        super(...args);
+
+        this.aliases      = ['bunny', 'bunnies'];
+        this.module       = 'Fun';
+        this.description  = 'Random Adorable Bunnies';
+        this.usage        = 'bunny';
+        this.example      = 'bunny';
+        this.cooldown     = 7500;
+        this.expectedArgs = 0;
+    }
+
+    async execute({ message }) {
+        try {
+			const utils = this.utils;
+			const errorText = `Error: ${this.config.emojis.rabbit || '🐰'} No bunnies found.`;

```

Comment:
just move this up 3 lines

Reply:
I am a bit confused, where is the error?

Reply:
What do you mean by move? I can't see an error.